src/docs/forms/components/CalendarInput/examples/06-datePicker.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import {CalendarInput} from "doj-react-adminlte/Form";
+
+export default class isSelectableDate extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            Calendarone: null,
+            Calendartwo: null
+        };
+    }
+
+    handleDateChange = (name, value) => {
+        this.setState({[name]: value});
+    };
+
+    render() {
+        const {Calendarone,Calendartwo} = this.state;
+
+        return (
+            <div className="row">
+                <div className="col-xs-6">
+                    <CalendarInput name="Calendarone" label="datePicker ='false' " value={Calendarone}
+                                   onChange={this.handleDateChange} datePicker={false}/>
+                </div>
+                <div className="col-xs-6">
+                    <CalendarInput name="Calendartwo" label="datePicker ='true'" value={Calendartwo}
```

Comment:
Instead of this, do a `timePicker` false. Adjust your labels correspondingly to show "Date picker only" and "Time picker only"